client/src/GlobalStyle.js

```diff
@@ -26,6 +26,7 @@ export default createGlobalStyle`
         line-height: 1.5;
         font-size: 1.1rem;
         font-family: sans-serif;
-        background-color: var(--background)
+        background-color: var(--background);
+        padding: 0;
```

Comment:
I think there is no browser that adds a default padding for `body`, so you don't need to remove it. You could instead remove the default `margin`, if you like.